### DISCUSSION TITLE
[@types/fluent-ffmpeg] .scale() method no typed in ScreenshotsConfig

### DISCUSSION THREAD

**Discussion by Ralex91** [DefinitelyTyped/DefinitelyTyped#68506] [Issues with a @types package]
The title says it all

Patch suggestion:
`/types/fluent-ffmpeg/index.d.ts`
```diff
interface ScreenshotsConfig {
    count?: number | undefined;
    folder?: string | undefined;
    filename?: string | undefined;
    timemarks?: number[] | string[] | undefined;
    timestamps?: number[] | string[] | undefined;
    fastSeek?: boolean | undefined;
    size?: string | undefined;
+   scale?: string | undefined;
}
```

[a comment by an automated account is not shown]

**Comment by defnot001** [comment]
Hello, after looking at the source code of node-fluent-ffmpeg, I can't find where the `scale` property would come from. The doc comment for the object starts [here](https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/blob/4e02d12573052c00ac4becee17e40580e855d308/lib/recipes.js#L79) and doesn't include a scale property. Could you point at where you found that config option?
